types/almost-equal/index.d.ts

```diff
@@ -0,0 +1,11 @@
+// Type definitions for almost-equal 1.1
+// Project: https://github.com/mikolalysenko/almost-equal#readme
+// Definitions by: Curtis Maddalozzo <https://github.com/cmaddalozzo>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function almostEqual(value: any, other: any, absoluteTolerance?: number, relativeTolerance?: number): boolean;
```

Comment:
According to the documentation, the first two arguments are numbers: "a and b are the two numbers to comapre [sic]"

Reply:
Good point. Fixed.